cms/djangoapps/contentstore/views/tests/test_tabs.py

```diff
@@ -52,7 +52,7 @@ def test_not_implemented(self):
             self.client.ajax_post(
                 self.url,
                 data=json.dumps({
-                    'tab_id_locator': {'tab_id': WikiTab.type},
+                    'tab_id_locator': {'tab_id': 'courseware'},
```

Comment:
The test was just verifying that the correct assertion is fired when trying to edit a tab. It doesn't matter which one, and I figured that the wiki is something we want to be able to remove in the future. I want to have all the wiki logic live in its own app, while for courseware that doesn't seem an issue.